src/python/clawutil/run_examples.py

```diff
@@ -0,0 +1,28 @@
+"""
+Run all examples in subdirectories of the directory from which this script
+is invoked, by doing `make all` in each.
+Also run any Jupyter notebooks to make .html versions.
+
+Two files
+    make_all_output.txt
+    make_all_errors.txt
+will be created that contain the output normally sent to the screen and any
+error messages from running the examples.
+"""
+
+from clawpack.clawutil import make_all
+import os
+
+env = os.environ
+env['GIT_STATUS'] = 'True'
+env['FFLAGS'] = '-O2 -fopenmp'
+env['OMP_NUM_THREADS'] = '6'
```

Comment:
I would suggest using these as defaults if they are not set in the environment whereas these are overriding the environment now.